src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/HardStopLicence.kt

```diff
@@ -134,6 +135,7 @@ class HardStopLicence(
   responsibleCom = responsibleCom,
 ),
   AlwaysHasCom,
+  ReviewablePreRelease,
```

Comment:
This isn't quite right, they can't be reviewed pre-release  - could maybe call it `Reviewable`